scripts/setup_linux.sh

```diff
@@ -169,16 +169,7 @@ if [[ "${_arg_modify_bashrc}" = on ]]; then
   echo -e "\n# Added by the garage installer" >> "${BASH_RC}"
```

Comment:
please try this script on a clean machine (or VM) and verify it works before submitting

Reply:
Done